dot/digest/digest_test.go

```diff
@@ -423,9 +422,35 @@ func TestHandler_HandleNextEpochData(t *testing.T) {
 
 	header := createHeaderWithPreDigest(t, 10)
 
+	finalisedCh := make(chan *types.FinalisationInfo)
+
+	handler := newTestHandler(t)
+	handler.finalised = finalisedCh
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	var wg sync.WaitGroup
+	wg.Add(1)
```

Comment:
if it's just one in the wait group, just close a signal channel `done := make(chan struct{})` instead

Same for below changes